modules/jetty-http-server/pom.xml

```diff
@@ -66,11 +66,6 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>ch.qos.logback</groupId>
-            <artifactId>logback-classic</artifactId>
```

Comment:
I removed logback-classic because we already had a logging binding from slf4j-test